src/current/cockroachcloud/serverless-faqs.md

```diff
@@ -1,180 +0,0 @@
-{{site.data.alerts.end}}
-
-### How do I start using CockroachDB {{ site.data.products.serverless }}?
-
-To get started with CockroachDB {{ site.data.products.serverless }}, <a href="https://cockroachlabs.cloud/signup?referralId=docs_serverless_faq" rel="noopener" target="_blank">sign up for a CockroachDB {{ site.data.products.cloud }} account</a>, click **Create Cluster**, then click **Create your free cluster**. Your cluster will be ready in 20-30 seconds. For more information, see [**Quickstart**]({% link cockroachcloud/quickstart.md %}).
-
-### What are the usage limits of CockroachDB {{ site.data.products.serverless }}?
-
-All non-contract CockroachDB {{ site.data.products.cloud }} organizations are given 50M RUs and 10 GiB of storage for free each month. Free resources do not apply to contract customers. Free resources can be spent across all CockroachDB {{ site.data.products.serverless }} clusters in an organization and will appear as a deduction on your monthly invoice.
-
-[Setting higher resource limits]({% link cockroachcloud/basic-cluster-management.md %}#edit-cluster-capacity) will allow your cluster to scale to meet your application's needs and maintain a high level of performance. If you reach your storage limit, your cluster will be throttled and you will only be able to delete data or increase your storage limit. If you reach your RU limit, your cluster will be disabled until the end of the billing cycle unless you increase your RU limit.
-
-Organizations without billing information on file can only create one cluster. Once you [set up billing information]({% link cockroachcloud/billing-management.md %}), your organization can have up to 200 clusters.
-
-### What is a Request Unit?
-
-{% include cockroachcloud/request-units.md %}
-
-### Do I have to pay for CockroachDB {{ site.data.products.serverless }}?
-
-No, you can create one CockroachDB {{ site.data.products.serverless }} cluster for free. The free cluster can use up to 50M RUs and 10 GiB of storage per month. If you need more resources, you can [set higher limits]({% link cockroachcloud/basic-cluster-management.md %}#edit-cluster-capacity) for your cluster. You will be charged only for your usage beyond the free threshold.
-
-### What regions are available for CockroachDB {{ site.data.products.serverless }} clusters?
-
-Refer to [CockroachDB {{ site.data.products.cloud }} Regions]({% link cockroachcloud/regions.md %}) for the regions where CockroachDB {{ site.data.products.dedicated }} and {{ site.data.products.serverless-plan }} clusters can be deployed. A multi-region Serverless cluster can have a maximum of six regions. To express interest in additional regions, [contact Support](https://support.cockroachlabs.com) or your Cockroach Labs account team.
-
-### How can I estimate how many RUs my workload will consume?
-
-{% include cockroachcloud/basic-usage.md %}
-
-### How can I reduce the number of RUs my workload consumes?
```

Comment:
Is this type of information collected elsewhere in the docs with reducing RU consumption as the main point? Or, maybe we don't care about this for 2.0?

Reply:
Yep, planning to move the content to the plan/manage cluster docs when I get to them -- made a note of it :)